src/languageServerInstaller.ts

```diff
@@ -9,16 +9,16 @@ import { Release, getRelease } from '@hashicorp/js-releases';
 export class LanguageServerInstaller {
 	public async install(directory: string): Promise<void> {
 		const { version: extensionVersion } = require('../package.json');
-		const lsVersionCmd = `${directory}/terraform-ls --version`;
 		const userAgent = `Terraform-VSCode/${extensionVersion} VSCode/${vscode.version}`;
-		let isInstalled = true;
+		let isInstalled = false;
 		try {
-			var { stdout, stderr } = await exec(lsVersionCmd);
-			var installedVersion = stdout || stderr;
+			var installedVersion = await getLsVersion(directory);
+			console.log(`Found installed LS ${installedVersion}`);
+			isInstalled = true
 		} catch (err) {
-			// TODO: verify error was in fact binary not found
-			isInstalled = false;
+			console.log(`failed to get version: ${err}`);
```

Comment:
Same as above – also we still aren't confirming that the error is because the language server isn't installed, so the comment could stay.

Reply:
True, but AFAIK this codepath is hit regularly on a fresh installation (_including_ when the binary is not found), so it's probably not something we'd want to notify the user about? I guess the best solution we'd be looking for eventually is to separate out upgrading and fresh installation and then treat this condition differently in each case?

Reply:
Yes, a new install is the most likely scenario, so we don't need to throw the error in this case. My preference is just to not leave in any logging that the user can't access.

Reply:
For some reason I thought `console.log` would appear in one of the Output panes, but it looks like I'm wrong. 😄 

Makes sense to leave it out then 👌🏻